src/AltinnCore/Designer/package-lock.json

```diff
@@ -1196,7 +1196,7 @@
     "@webassemblyjs/helper-buffer": {
       "version": "1.5.12",
       "resolved": "https://registry.npmjs.org/@webassemblyjs/helper-buffer/-/helper-buffer-1.5.12.tgz",
-      "integrity": "sha512-tJNUjttL5CxiiS/KLxT4/Zk0Nbl/poFhztFxktb46zoQEUWaGHR9ZJ0SnvE7DbFX5PY5JNJDMZ0Li4lm246fWw==",
```

Comment:
Sha should be sha512: https://stackoverflow.com/questions/47638381/why-did-package-lock-json-change-the-integrity-hash-from-sha1-to-sha512

Reply:
Ok, any idea why my package-lock,json has been changed back? I discarded all changes to these files and did a new install/build before pushing these files. Seems to be ok for the dashboard file.

Reply:
Don't know...